source/_integrations/firefly_iii.markdown

```diff
@@ -0,0 +1,48 @@
+---
+title: Firefly III
+description: Instructions on how to integrate Firefly III with Home Assistant.
+ha_category:
+  - Sensor
+ha_release: "2025.11"
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@erwindouna'
+ha_domain: firefly_iii
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
```

Comment:
```suggestion
ha_integration_type: hub
```
I think in core it is defined as hub
https://github.com/home-assistant/core/pull/147062/files#diff-604d8c76dd6c7a5d1d54d3629eca60df0188f45cf8243d87791ce338f500bbffR1983